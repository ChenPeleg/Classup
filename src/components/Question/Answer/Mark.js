import styles from "./Mark.module.scss";
import React from "react";
import Vmark from "../../../assets/images/check-mark.svg";
import Xmark from "../../../assets/images/x-mark.svg";

const mark = (props) => {
  return (
    <div className={styles.checkMarkWrapper}>
      <img
        src={props.marked_Answer === "RIGHT" ? Vmark : Xmark}
        className={styles.checkMark}
        alt="MyBurger"
      />
    </div>
  );
};
export default mark;
