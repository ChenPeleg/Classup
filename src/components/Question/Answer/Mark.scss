.check-mark-wrapper {
  position: absolute;
  width: 50px;
  height: 50px;
  top: -10px;
  left: -10px;
}
.check-mark {
  position: relative;
  width: 100%;
  height: 100%;
  opacity: 0;
  animation: check-animate 0.3s forwards;
}
@keyframes check-animate {
  0% {
    opacity: 0;
  }
  50% {
    opacity: 0.5;
  }
  100% {
    opacity: 1;
  }
}
