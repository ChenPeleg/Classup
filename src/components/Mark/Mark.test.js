import React from "react";
import Mark from "./Mark";
import { shallow } from "enzyme";

const mark = shallow(<Mark />);

describe("<Mark>", () => {
  it("renders correctly", () => {
    expect(mark).toMatchSnapshot();
  });
});
