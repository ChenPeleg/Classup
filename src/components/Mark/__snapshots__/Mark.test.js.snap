// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Mark> renders correctly 1`] = `ShallowWrapper {}`;
